__docs__/wonder-blocks-tokens/semantic-color-groups.stories.tsx

```diff
@@ -177,163 +192,25 @@ export const LearningShadow = () => {
 };
 
 export const Khanmigo = () => {
-    return <ColorGroup colors={semanticColor.khanmigo} group="khanmigo" />;
-};
-
-export const Mastery = () => {
-    return <ColorGroup colors={semanticColor.mastery} group="mastery" />;
-};
-
-export const ActionPrimaryProgressive = () => {
```

Comment:
Moved the examples using deprecated tokens to `deprecated-semantic-colors.stories.tsx`